### PR TITLE
Fix GET naming scheme and table manipulation script

### PR DESCRIPTION
**Pull request by ilikesymmetry**
Modularizes the GET naming fixes and catches more locations.  Also wrote a script to manually change the formatting of our start and end times to help out frontend.

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -11,13 +11,21 @@
 DINING_STACK_URL = 'https://raw.githubusercontent.com/cuappdev/DiningStack/master/DiningStack/'
 EATERY_DATA_PATH = './eatery-data/'
 GET_URL = 'https://services.get.cbord.com/GETServices/services/json'
+GET_LOCATIONS = {
+    'Attrium Cafe': 'Atrium Café',
```
Comment (kq-li) [inline src/constants.py:15]: is this supposed to have 2 t's?

**Review by kq-li — APPROVED** [review]
lgtm!

File: src/constants.py (hunk 1 above)
Comment (ilikesymmetry) [reply src/constants.py:15]: no, that's why we change it

Comment (kq-li) [reply src/constants.py:15]: ahh thought it was a typo, all good

**Review by orkosinha — APPROVED** [review]
lgtm!

**Pull request merged**